tripper/utils.py

```diff
@@ -234,10 +234,16 @@ def parse_literal(literal: "Any") -> "Any":
     ):
         datatype = str(literal.datatype)
 
-    # This will handle rdflib literals correctly and probably most other
-    # literal representations as well.
+    # This should handle rdflib literals correctly (and probably most other
+    # literal representations as well)
     if hasattr(literal, "value"):
```

Comment:
But can a literal not have the attribute? Isn't it always there, but sometimes with value None? (I thought the reason for this fix is exactly that)

Reply:
Agree, the argument name `literal` is not very well chosen. It was changed by Casper from `value` to `literal` in one of his early reviews.